fixtures/ext-types/guid/src/lib.rs

```diff
@@ -0,0 +1,68 @@
+use uniffi::FfiConverter;
+
+// A trivial guid.
+pub struct Guid(pub String);
+
+fn get_guid(guid: Option<Guid>) -> Guid {
+    match guid {
+        Some(guid) => guid,
+        None => Guid("NewGuid".to_string()),
+    }
+}
+
+pub struct GuidHelper {
+    pub guid: Guid,
+    pub guids: Vec<Guid>,
+}
+
+fn get_guid_helper(vals: Option<GuidHelper>) -> GuidHelper {
+    match vals {
+        None => GuidHelper {
+            guid: Guid("first-guid".to_string()),
+            guids: vec![
+                Guid("second-guid".to_string()),
+                Guid("third-guid".to_string()),
+            ],
+        },
+        Some(vals) => vals,
+    }
+}
+
+// The name is hard-coded based on the 2 types involved.  Definitely open to a suggestions for how
+// we generate the name.
+struct FFIConverterWrappingGuidString;
```

Comment:
Based on b5e7836dcc429a43a5a7a72879a985c6bddfa2f9, the canonical name for (almost) all types can be deduced knowing just its name and without knowing its type - this means we can now just `use` an external type without knowing what is in that remote idl.